### DISCUSSION TITLE
Next game line?

### DISCUSSION THREAD

**Discussion by uSlackr** [vasqued2/ha-teamtracker-card#24] [Ideas]
I said this in a previous post - I love this integration. Thanks again for your time. It brings me joy!
Idea or question.  On the card, would it be possible to show the next game all the time?  For hockey games for example, it shows the results from yesterday, but no indication when the next game is.  Having that on the bottom line pre or post game would be helpful. Maybe this is already possible?
![image](https://user-images.githubusercontent.com/25751399/204149065-3e865e5d-46b0-48ad-924b-8a1da912ee37.png)

**Comment by vasqued2** [comment]
Very rarely does the API return more than a single game for a team.

When it does, I use the completed game until the API drops it or until the upcoming game is within 18 hours of starting, at which point I use the upcoming game.

Here is a link to an article that describes how someone is using Google Calendar to display info on upcoming games.  See Step 3.1.  Looks like what you are trying to do.

[https://smarthomepursuits.com/teamtracker-integration-home-assistant-real-time-sports-scores/](https://smarthomepursuits.com/teamtracker-integration-home-assistant-real-time-sports-scores/)
